Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -222,17 +254,17 @@ var user = userService.getCurrentUser().then(function (user) {
 Finally we can now update our view to incorporate the current user's name in our Welcome Message:
 
 ```html
-<h1>Welcome {{vm.UserName}} ...to Umbraco</h1>
+<h1>Welcome {{vm.UserName}} to Umbraco</h1>
 ```
 
-![Custom Dashboard Welcome Message With Current User's Name](images/welcomemessagepersonalised.jpg)
+![Custom Dashboard Welcome Message With Current User's Name](images/welcomemessagepersonalised-v8.png)
 
 and for reference the full contents of /customwelcomedashboard.controller.js at this stage of the tutorial should look like this:
 
 ```js
 angular.module("umbraco").controller("CustomWelcomeDashboardController", function ($scope,userService) {
     var vm = this;
-    vm.UserName = 'guest';
```

Comment:
This code example, is the exact same snippet as the one on lines 204 to 248, isn't it?
Is it necessary to have it two times in a row? 🤔